tensorflow/lite/kernels/elementwise_test.cc

```diff
@@ -123,7 +148,7 @@ TEST(ElementWise, Log) {
   EXPECT_THAT(m.GetTensorShape(m.output()), ElementsAreArray({1, 1, 4, 1}));
 }
 
-TEST(FloatActivationsOpTest, Abs) {
+TEST(ElementWise, Abs) {
```

Comment:
I renamed `FloatActivationsOpTest` to `ElemWise` for `Abs` and `AbsInt8` for coherence with `AbsInt16`. 

In `acceleration_test_list.cc` the Abs test is mentioned as `ElementWise/Abs` so I suppose the `FloatActivationsOpTest` was a mistake but it'd be good to make sure of it.

Reply:
Similarly, if my understanding is correct, it appears that the change in [this line](https://github.com/tensorflow/tensorflow/pull/43098/files#diff-ffc98f167365cff2837139e9af87b3deR724) is a fix to a small error from an earlier commit.